examples/decode_sensor_binary_log.py

```diff
@@ -205,6 +205,60 @@ def parser(self):
                 # else message is still being parsed
 
 
+@dataclass(init=False, order=True)
+class Ping1DSettings:
+    transmit_duration : int # [us]
+    scan_start        : int # [mm] from transducer
+    scan_length       : int # [mm]
+    gain_setting      : int # [0-6 -> 0.6-144]
+
+    def __init__(self, profile: PingMessage):
+        for attr in self.__annotations__:
+            setattr(self, attr, getattr(profile, attr))
+
+    @property
+    def gain(self):
+        """ Returns device receiver 'gain', as specified by 
+        https://docs.bluerobotics.com/ping-protocol/pingmessage-ping1d/#1300-profile.
+        
+        """
+        return (0.6, 1.8, 5.5, 12.9, 30.2, 66.1, 144)[self.gain_setting]
+
+
+@dataclass(init=False, order=True)
+class Ping360Settings:
+    mode               : int = 1 # Ping360 = 1
+    gain_setting       : int = 0 # 0-2 (low,normal,high)
+    transmit_duration  : int     # 1-1000 [us]
+    sample_period      : int     # 80-40_000 [25ns]
+    transmit_frequency : int     # [kHz]
+    number_of_samples  : int
+
+    def __init__(self, device_data: PingMessage):
+        for attr in self.__annotations__:
+            setattr(self, attr, getattr(device_data, attr))
+
+    @property
+    def gain(self):
+        return ('low', 'medium', 'high')[self.gain_setting]
+
+    @property
+    def sample_period_us(self):
+        """ Returns device sample period in microseconds. """
+        return self.sample_period * 25e-3
+    
+    def meters_per_sample(self, v_sound):
+        """ Returns the distance [m] covered by each sample of a ping.
+
+        Distance depends on 'v_sound' [m/s], the speed of sound in the
+            surrounding liquid.
+
+        """
+        # time of flight -> v_sound * (there + back) / 2
+        #  -> v_sound * us * 1e-6 / 2
+        return v_sound * self.sample_period_us * 5e-7
```

Comment:
Isn't it better to let the / 2 dividing on the code and remove the above comment ?
This looks like an unnecessary performance improvement.